reana_server/fetcher.py

```diff
@@ -236,11 +237,11 @@ def get_fetcher(url: str, output_dir: str) -> WorkflowFetcherBase:
 
     if extension == ".git":
         return WorkflowFetcherGit(url, output_dir)
+    elif extension == ".zip":
+        return WorkflowFetcherZip(url, output_dir, archive_name=basename)
```

Comment:
Hmm, we may want to refactor the source guessing later down the line, since "zip" can be hosted on URL, on GitHub... So I see the if/elif/elif chain order can be fragile. Basically, we could consider URL-fetcher and GitHub-fetcher as two main "source fetcher" methods, and Zip only as a kind of post-processor of what was fetched from wherever... CC @mdonadoni for later.